### PR TITLE
cucumber scenario

### PR DESCRIPTION
**Pull request by YevheniiChornyi**
idk what exactly i should do in scenario, so im waiting for clarifications about best practices

### CODE REVIEW

File: src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package yevhenii.lostfilmdemo.controllers;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.context.annotation.Profile;
+
+@Profile("test")
+@FeignClient(url = "http://localhost:8089/www.omdbapi.com/", name = "omdb")
```
Comment (mmarusich) [inline src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java]: you don't need to create separate client, you can put url into properties file and place here ${property.name}. Then just ovverider property in test properties

File: src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            .withDatabaseName("lostfilm")
+            .withUsername("root")
+            .withPassword("root2021");
+
+    @Given("I go mock tv series sites")
+    public void iGoMockTvSeriesSites() {
+        wiremockStub();
+    }
+
+    @When("I go to find new series")
```
Comment (mmarusich) [inline src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java]: `I start scheduled process to retrieve series from Lostfilm`

File: src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    public void iGoToLostfilmRss() throws SchedulerException, InterruptedException {
+        scheduler.triggerJob(new JobKey("lostfilm_job", Scheduler.DEFAULT_GROUP));
+        Thread.sleep(5000);
```
Comment (mmarusich) [inline src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java:56]: extract this to separate step `I wait scheduled process to complete`

File: src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    @Autowired
+    private TVRecordConvertor convertor;
+
+    @ClassRule
+    private final MySQLContainer<?> mySQLContainer = new MySQLContainer<>(DockerImageName.parse("mysql:8.0.24"))
+            .withDatabaseName("lostfilm")
+            .withUsername("root")
+            .withPassword("root2021");
+
+    @Given("I go mock tv series sites")
```
Comment (mmarusich) [inline src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java]: I mocking Lostfilm RSS feed response

File: src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+        Thread.sleep(5000);
+    }
+
+    @Then("I should get TVSeries")
+    public void iShouldGetTVSeries() {
+        verify((getRequestedFor(urlEqualTo("/www.lostfilm.tv/rss.xml"))));
+    }
+
+
+    @And("put tvSeries into db:")
```
Comment (mmarusich) [inline src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java]: I except following services in DB

File: src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        AtomicInteger count = new AtomicInteger();
+        repository.readAll().stream()
+                .map(convertor::convert)
+                .peek(a -> a.setImdbEpisode(null))
+                .forEach(a -> assertThat(a)
+                        .isEqualTo(seriesList.get(count.getAndIncrement())));
+
+    }
+
+    @And("send message, that we should find info from imdb and also put updated tvSeries to db")
```
Comment (mmarusich) [inline src/test/java/yevhenii/lostfilmdemo/cucumber/TVSeriesSteps.java]: I expect following ImDB info to be populated

**Review by mmarusich — APPROVED** [review]
LGTM, few small comments

File: src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java (hunk 1 above)
Comment (YevheniiChornyi) [reply src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java]: idk how to use @Value in interface

Comment (mmarusich) [reply src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java]: you don't need to use value interface, you should have `(url = "${omdb.api.url}")`

**Pull request merged**